test/Log/LogServiceTest.php

```diff
@@ -115,39 +115,39 @@ public function testSetPath()
     }
 
     /**
-     * Test getting configured default log level.
+     * Test getting configured default log priority.
      */
-    public function testGetDefaultLogLevel()
+    public function testGetDefaultLogPriority()
```

Comment:
I was thinking we would get rid of the 'Log' bit in the names, since it should be clear from the context. Thank you!